app/components/BackButton/BackButton.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react'
+import Arrow from '../../assets/icons/arrow.svg'
+import styles from '../Button/Button.scss'
+
+type Props = {
+  className: ?string,
+  routeTo: string,
+  history: {
+    push: Function
+  }
+}
+
+export default class BackButton extends React.Component<Props> {
+  render = () => {
+    return (
+      <span className={this.props.className} onClick={this.handleClick}>
```

Comment:
Seems like we can/should just make this a link, e.g.:

```jsx
import { Link } from 'react-router-dom'

render = () => {
  return (
    <Link to={this.props.routeTo}>
      <Arrow className={styles.icon} />
    </Link>
  )
}
```

That will also make it so we can remove the `history` propType.

Reply:
👍